packages/web/docs/src/pages/docs/gateway/index.mdx

```diff
@@ -83,6 +83,11 @@ environment. Then, you can install Hive Gateway CLI with your preferred package
 npm i @graphql-hive/gateway
 ```
 
+<Callout>
```

Comment:
I think we still don't have a clear recommendation here regarding Bun. It's new and cool, but I feel like it's still not in the same state of Node when it comes to stability. 

I think maybe phrasing it as something like "Hive Gateway is runtime-agnostic and on different runtimes. The Guild's development process uses NodeJS, and the gateway can also run on alternative runtimes like Bun"?

Reply:
We have both unit and E2E tests with Bun. So Bun is tested exactly in the same way Node is tested in the CI. We can rephrase the sentence here but the idea here is to emphasize that "You can use Hive Gateway CLI on Bun just like you do on Node".

Reply:
what is the reason to specifically callout Bun here?

Reply:
We refer `bun install` with `npm2yarn` in our docs, and in the other parts of this section, it looks like CLI only supports Node.js but it is incorrect. CLI also supports Bun as we test it e2e and run unit tests against its test runner.
Do you think this gives a wrong message?

Reply:
is it also true for other environments?  like Deno?

Reply:
ok I see, thanks, approved by me